pkg/proxy/tcp_proxy.go

```diff
@@ -8,12 +8,14 @@ import (
 	"github.com/Sirupsen/logrus"
 )
 
+// TCPProxy is a proxy for TCP connections.
```

Comment:
I'm tempted to say something like on ``TCPListener`` (http://golang.org/src/net/tcpsock_posix.go?s=7230:7268#L221) as it's mostly meant to be used as ``proxy.NewProxy`` and the function will figure out if it should use ``TCP`` or ``UDP``.

> As a resource, does it require management?

Not sure at all about that :sweat:.